js/preprocessor.js

```diff
@@ -207,11 +212,11 @@ YoastSEO_PreProcessor.prototype.replaceDiacritics = function( textString ) {
  * input.
  *
  * @param inputString
- * @returns {yst_preProcessor}
+ * @returns {YoastSEO.PreProcessor}
  */
-YoastSEO_preProcessor = function( inputString ) {
+YoastSEO.createPreProcessor = function( inputString ) {
```

Comment:
This sort of mimics Singletonish behaviour. Maybe `getPreProcessor` is more appropriate?